.travis.yml

```diff
@@ -70,25 +70,14 @@ install:
 - pip install --quiet xlrd
 - pip install --quiet openpyxl
 #
-- if [ -n "${EXTRAS}" ]; then conda install -q -y --channel conda-forge pyomo.extras; fi
-- if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-     if [ -n "${EXTRAS}" ]; then
-        pip install scipy;
-     fi
-  fi
-- conda install -q -y --channel conda-forge glpk
-- glpsol -v
-- conda install -q -y --channel cachemeorg ipopt_bin
-- ipopt -v
-#
 # Install free version of GAMS: executable and API bindings
 # No explicit API version for python 3.5, use 3.4
 #
 - wget https://d37drm4t2jghv5.cloudfront.net/distributions/24.8.5/linux/linux_x64_64_sfx.exe
 - chmod u+x linux_x64_64_sfx.exe
 - ./linux_x64_64_sfx.exe > /dev/null
 - PATH=${PATH}:$PWD/gams24.8_linux_x64_64_sfx
-# Change default NLP solver to baron for testing
+# Change default NLP solver to baron for testing, performs better than CONOPT
```

Comment:
Nick will be happy, but just so that you know, BARON uses either CONOPT, IPOPT, FilterSQP, or some other NLP solver for NLP problems. Maybe it's the pre-solve that they have.